servicetalk-transport-netty-internal/src/main/java/io/servicetalk/transport/netty/internal/NettyConnectionContext.java

```diff
@@ -56,7 +56,9 @@
     Single<Throwable> transportError();
 
     /**
-     * Returns a {@link Completable} that notifies when the connection has begun its closing sequence.
+     * Returns a {@link Completable} that notifies when the connection has begun its closing sequence. The
+     * {@code Completable} is not required to be blocking-safe and should be offloaded if the
```

Comment:
Consider writing it as an additional note instead of extending the first sentence:
```
<p>
<b>Note:</b> The {@code Completable} is not required...
```
The first sentence is used as a short summary when javadoc is rendered.

Reply:
The additional text is a second sentence. I can move it to a note though to make the requirement stand out.

Reply:
You are right, I though that it's everything before `<p>` because Intellij IDEA renders it in that way.